tests/gdx-tests/src/com/badlogic/gdx/tests/g3d/ParticleControllerInfluencerSingleTest.java

```diff
@@ -0,0 +1,197 @@
+
```

Comment:
This file demonstrates the correct behavior when the rotationChannel is properly updated and exhibits incorrect behavior without that change. 
In particular, the ParticleController is correctly rotated when the SpawnInfluencer properly accounts for rotation.